engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/task/IdentityLinkDto.ftl

```diff
@@ -0,0 +1,21 @@
+<@lib.dto required = ["type"] >
+
+    <@lib.property
+        name = "userId"
+        type = "string"
+        desc = "The id of the user participating in this link. Either `userId` or `groupId` is set." />
+
+    <@lib.property
+        name = "groupId"
+        type = "string"
+        desc = "The id of the group participating in this link. Either `groupId` or `userId` is set." />
+
+    <@lib.property
+        name = "type"
+        type = "string"
+        enumValues = ['"assignee"', '"candidate"', '"owner"']
+        last = true
+        desc = "The type of the identity link. Can be any defined type. `assignee` and `owner`
```

Comment:
I'll use this line to reply to the general review comment:
Yes, the `IdentityLinkDto` properties are documented in three different ways in the endpoints. That's why I decided to generalize them. This one varies the most.

* As I mentioned, I will expand the description, by adding the default engine relationships (and remove the `enumValues`).

Regarding the `required` type, we actually do validate the presence of the property, it's just buried pretty deep in the code :smile:. You can find the validations here:
* https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/impl/cmd/AddIdentityLinkCmd.java#L60
* https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/impl/cmd/DeleteIdentityLinkCmd.java#L61